specification/src/main/asciidoc/platform/ApplicationProgrammingInterface.adoc

```diff
@@ -30,38 +30,35 @@ might nonetheless include support for the technology. In such a case,
 the requirements for that technology described in this chapter would
 apply.
 
+[[a3537]]
 ==== Java Compatible APIs
 
 The containers provide all application
 components with at least the Java Platform, Standard Edition, v8 (Java
 SE) APIs. Containers may provide newer versions of the Java SE platform,
-provided they meet all the Jakarta EE platform requirements. The Java SE
-platform includes the following enterprise technologies:
+provided they meet all the Jakarta EE platform requirements. If the container
+provides a version of the Java SE platform newer than Java SE 8 and that
+version no longer includes one or more APIs that were available in Java SE 8,
+the container is not required to provide those APIs, unless they are
+one of the enterprise technologies listed in this specification.
```

Comment:
I'm stumbling over the ", .. unless they are one of the ...".  I read "they" as referring to the container.  But, I believe you mean "they" refers to the missing APIs...  And, this whole sentence is already getting pretty long.  I don't have better wording off the top of my head, but I think this paragraph needs to be adjusted.

Reply:
I added a commit where I refactor this section quite a bit. The wording could still use some improvement but I tried to change the structure a bit to hopefully clarify things.

Note that I dropped the listing of JAX-WS/JAXB/JAF/SAAJ/Annotations in the SE section and just noted that some things SE 8 provided are now provided by Jakarta. The idea being to focus the section on the stuff that isn't a Jakarta spec.